test/dogma/script_test.exs

```diff
@@ -167,8 +176,8 @@ defmodule Dogma.ScriptTest do
     end
 
     should "run the tests", context do
```

Comment:
Can we change this test name (and maybe the with block if required) to explain more clearly what exactly is going on here? i.e. What's the relationship with the custom rules and the CustomRules module.